src/client/index.ts

```diff
@@ -0,0 +1,56 @@
+import { NotifyResponse, Waiter } from "../types";
+import { Client as QStashClient } from "@upstash/qstash";
+import { makeGetWaitersRequest, makeNotifyRequest } from "./utils";
+
+type ClientConfig = ConstructorParameters<typeof QStashClient>[0];
+
+export class Client {
```

Comment:
It would be really good if there was a docstring showing an example snippet using this client. And maybe the .fromEnv() pattern that we have in other clients

Reply:
added docstring

we don't have the fromEnv utility in qstash-js, I think we can add them but let's do it later